ledger/participant-state/kvutils/tools/src/test/scala/ledger/participant/state/kvutils/tools/integritycheck/StateUpdateComparisonSpec.scala

```diff
@@ -68,22 +65,10 @@ final class StateUpdateComparisonSpec
       )
       val right =
         aTransactionAcceptedUpdate.copy(blindingInfo = Some(blindingInfo))
+      val normalizers = List(BlindingInfoNormalizer)
 
       ReadServiceStateUpdateComparison
-        .compareUpdates(left, right, NormalizationSettings(ignoreBlindingInfo = true))
-        .map(_ => succeed)
-    }
-
-    "ignore transaction ID for TransactionAccepted updates" in {
```

Comment:
Why has this test case been removed?

Reply:
I decided to drop the support for transaction id normalization, as my impression is that it was only used in the integration kit repo. It has been replaced there with the mapping-based ID generation i.e. the IDs are rewritten anyway. I'm waiting for the test results to confirm it.

Reply:
Yeah, the tests, are green. Please let me know if you think it would be worth keeping for manual integrity checks.

Reply:
I don't know if there is any use case left for it; if not, I agree about removing it. In case it is needed again, we can always re-instate it.